modules/3d-tiles/test/data/index.json

```diff
@@ -0,0 +1,1177 @@
+{
+  "Batched": {
```

Comment:
Do we need such a complicated hierarchical list? Isn't it enough with just a list of the filenames?

Reply:
Some examples have 2 files, one is `tileset.json` and the other is a binary file. Other examples have more files, I am thinking we need all the files processed in order to see the complete example.

Reply:
Oh I see. The `tileset.json` should contain the URLs or at least filenames of all the other files (assuming they are tiles).

 If that is the case, maybe we only need a list of tileset.json files...